docs/tutorials/tutorial_3k_tcr.md

```diff
@@ -507,38 +508,38 @@ ir.pl.spectratype(
 
 ### Repertoire overlaps
 
-```python
-df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
-```
+Overlaps in the adaptive immune receptor repertoire of samples or sample groups enables to pinpoint important clonotype groups, as well as to provide a measure of similarity between samples.  
+Running Scirpy's `repertoire_overlap` tool creates a matrix featuring the abundance of clonotypes in each sample. Additionally, it also computes a (Jaccard) distance matrix of samples as well as the linkage of hierarchical clustering.
 
 ```python
+df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
 df.head()
 ```
 
-```python
-ir.pl.repertoire_overlap(adata, "sample")
-```
+The distance matrix can be shown as a heatmap, while samples are reordered based on the linkage computed previously.
```

Comment:
It's not really using the linkage from the cell above (`inplace=False`). How about 
"reordered based on hierarchical clustering"?